docs/api/context-bridge.md

```diff
@@ -0,0 +1,129 @@
+# contextBridge
+
+> Create a safe, bi-directional, synchronous bridge across isolated contexts
+
+Process: [Renderer](../glossary.md#renderer-process)
+
+An example of exposing an API to a renderer from an isolated preload script is given below:
+
+```javascript
+// Preload (Isolated World)
+const { contextBridge, ipcRenderer } = require('electron')
+
+contextBridge.exposeInMainWorld(
+  'electron',
+  {
+    doThing: () => ipcRenderer.send('do-a-thing')
+  }
+)
+```
+
+```javascript
+// Renderer (Main World)
+
+window.electron.doThing()
+```
+
+## Glossary
+
+### Main World
+
+The "Main World" is the javascript context that your main renderer code runs in.  By default the page you load in your renderer
+executes code in this world.
+
+### Isolated World
+
+When `contextIsolation` is enabled in your `webPreferences` your `preload` scripts run in an "Isolated World".  You can read more about
+context isolation and what it affects in the [BrowserWindow](browser-window.md) docs.
+
+## Methods
+
+The `contextBridge` module has the following instance methods:
+
+### `contextBridge.exposeInMainWorld(apiKey, api)`
+
+* `apiKey` String - The key to inject the API onto `window` with.  The API will be accessible on `window[apiKey]`.
```

Comment:
nit: I think we can do better than `apiKey`, which is avoidably loaded.

I'm having trouble envisioning when a consumer would want to expose multiple API objects on `window` in the main world. It seems more likely that they'll want to expose a single object under one key, almost every time. Would it make sense to include a reasonable default and make the API slightly simpler? Maybe a signature like:
```ts
contextBridge.exposeInMainWorld(
  api: { [key: string]: ContextBridgeSerializeable },
  name?: string = 'electron'
);
``` 
And your example becomes:
```ts
contextBridge.exposeInMainWorld({
  doThing: () => ipcRenderer.send('do-a-thing')
})
```

Reply:
I've got a few use cases in mind for exposing multiple things it helps in tests + actual apps might want to use it (including us 😄 ).  Not sure I want to standardize `window.electron`